README.md

```diff
@@ -161,15 +161,15 @@ All Graph Neural Network layers are implemented via the **[`nn.MessagePassing`](
 A GNN layer specifies how to perform message passing, *i.e.* by designing different message, aggregation and update functions as defined [here](https://pytorch-geometric.readthedocs.io/en/latest/tutorial/create_gnn.html).
 These GNN layers can be stacked together to create Graph Neural Network models.
 
-- **[GCNConv](https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.nn.conv.GCNConv.html)** from Kipf and Welling: [Semi-Supervised Classification with Graph Convolutional Networks](https://arxiv.org/abs/1609.02907) (ICLR 2017) \[[**Example**](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/gcn.py)\]
-- **[ChebConv](https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.nn.conv.ChebConv.html)** from Defferrard *et al.*: [Convolutional Neural Networks on Graphs with Fast Localized Spectral Filtering](https://arxiv.org/abs/1606.09375) (NIPS 2016) \[[**Example**](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/gcn.py#L36-L37)\]
-- **[GATConv](https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.nn.conv.GATConv.html)** from Veličković *et al.*: [Graph Attention Networks](https://arxiv.org/abs/1710.10903) (ICLR 2018) \[[**Example**](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/gat.py)\]
+- **[GCNConv](https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.nn.conv.GCNConv.html)** from Kipf and Welling: [Semi-Supervised Classification with Graph Convolutional Networks](https://arxiv.org/abs/1609.02907) (ICLR 2017) \[[**Example1**](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/gcn.py), [**Example2**](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/planetoid_train.py)\]
+- **[ChebConv](https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.nn.conv.ChebConv.html)** from Defferrard *et al.*: [Convolutional Neural Networks on Graphs with Fast Localized Spectral Filtering](https://arxiv.org/abs/1606.09375) (NIPS 2016) \[[**Example1**](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/gcn.py#L36-L37), [**Example2**](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/planetoid_train.py#L115-L116)\]
+- **[GATConv](https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.nn.conv.GATConv.html)** from Veličković *et al.*: [Graph Attention Networks](https://arxiv.org/abs/1710.10903) (ICLR 2018) \[[**Example1**](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/gat.py), [**Example2**](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/planetoid_train.py)\]
```

Comment:
rather than showing planetoid_train in all these duplicate places, please explain that "planetoid_train.py offers a unified codebase covering GCN, GAT, and Cheb Convs, as well as many more"